task/sql-tasks.js

```diff
@@ -281,9 +466,29 @@ async function task_1_21(db) {
  * @return {array}
  */
 async function task_1_22(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+    SELECT
+        cust.CompanyName,
+        prod.ProductName as "ProductName",
+        ord_det.UnitPrice as "PricePerItem"        
+    FROM Customers AS cust
+    JOIN Orders AS ord ON ord.CustomerID = cust.CustomerID
+    JOIN OrderDetails AS ord_det ON ord_det.OrderID = ord.OrderID
+    JOIN Products AS prod ON prod.ProductID = ord_det.ProductID
+    WHERE ord_det.UnitPrice = (

```

Comment:
You can move the complex Where condition as part of SELECT and avoid DISTINCT function also